filebeat/tests/system/test_syslog.py

```diff
@@ -161,15 +177,9 @@ def test_syslog_with_unix(self):
 
             self.wait_until(lambda: self.log_contains("Started listening for UNIX connection"))
 
-            sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)  # UNIX
-
-            sock.connect(path)
-
-            for n in range(0, 2):
-                m = "<13>Oct 11 22:14:15 wopr.mymachine.co postfix/smtpd[2000]:" \
-                    " 'su root' failed for lonvick on /dev/pts/8 {}\n"
-                m = m.format(n)
-                sock.send(m.encode("utf-8"))
+            sock = send_over_socket(path,
+                                    "<13>Oct 11 22:14:15 wopr.mymachine.co postfix/smtpd[2000]:"
+                                    " 'su root' failed for lonvick on /dev/pts/8 {}\n")
```

Comment:
Maybe we want to send more than one event in this test?

Reply:
Every test sends three events with the same message ending with the index of that message. You mean that we should send a different message every time we send an event?

Reply:
Yes. But also consider to send very big messages that might not fit the read buffer. Test the edge cases. Plus if something fails it should be more obvious which exact message failed or was missing.